Adapter.C

```diff
@@ -186,6 +190,14 @@ bool preciceAdapter::Adapter::configFileRead()
             }
         }
 
+        // If the Volume_Coupling module is enabled, create it, read the
+        // Volume_Coupling-specific options and configure it.
+        if (Volume_Couplingenabled_)
```

Comment:
May I ask you to add this option in the `if` block below in order to bail out in case no module was enabled?